checker/src/org/checkerframework/checker/index/upperbound/UpperBoundAnnotatedTypeFactory.java

```diff
@@ -76,7 +76,7 @@
 
     public final AnnotationMirror UNKNOWN, BOTTOM, POLY;
 
-    private final IndexMethodIdentifier imf;
+    protected final IndexMethodIdentifier imf;
```

Comment:
Why is this `protected` instead of package-private (no modifier)? It's not used anywhere outside the package, and package-private prevents subclasses outside the package from seeing the field. I'm not sure why anyone would want to subclass `UpperBoundAnnotatedTypeFactory`, but we can expose this if there's a good reason.